src/core_modules/capture-ui/FlatList/FlatList.component.js

```diff
@@ -1,13 +1,47 @@
 // @flow
-import React from 'react';
-import uuid from 'uuid/v4';
+import React, { type ComponentType } from 'react';
+import cx from 'classnames';
+import { colors } from '@dhis2/ui';
+import { withStyles } from '@material-ui/core';
 import type { Props } from './flatList.types';
 
-export const FlatList = ({ list, className, dataTest, renderItem }: Props) => (
-    <div
-        data-test={dataTest}
-        className={className}
-    >
-        {list.map(item => (<div key={uuid()}>{renderItem(item)}</div>))}
-    </div>
-);
+
+const styles = {
+    flatListWrapper: {
+        padding: '0 16px',
```

Comment:
Hey, 
What do you think about using the constant `spacersNum.dp16` from `@dhis2/ui` instead of the number 16 to help maintain the homogeneity of the stypes?
Thank you!

Reply:
@simonadomnisoru totally makes sense! I will change that